datafusion/sqllogictest/test_files/spark/array/shuffle.slt

```diff
@@ -22,7 +22,7 @@ SELECT array_sort(shuffle([1, 2, 3, 4, 5, NULL])) = [NULL,1, 2, 3, 4, 5];
 true
 
 query B
-SELECT shuffle([1, 2, 3, 4, 5, NULL]) != [1, 2, 3, 4, 5, NULL];
+SELECT shuffle([1, 2, 3, 4, 5, NULL], 1) != [1, 2, 3, 4, 5, NULL];
 ----
 true
```

Comment:
We can just assert the output list instead of it being an inequality with its sorted version now, e.g.

```sql
query ?
SELECT shuffle([1, 2, 3, 4, 5, NULL], 1);
----
[2, 5, NULL, 3, 4, 1]
```